frontend/src/containers/AddScholarshipViaCSVPage/AddScholarshipViaCSVPage.tsx

```diff
@@ -0,0 +1,160 @@
+                    </Link>
+
+                    <Typography
+                        p={'40px 0 30px'}
+                        sx={{
+                            fontFamily: 'Roboto',
+                            fontSize: '48px',
+                            fontWeight: '700',
+                            color: '#002147'
+                        }}
+                    >
+                        Add New Scholarship via CSV
+                    </Typography>
+
+                    <Box
+                        p={'40px'}
+
+                        sx={{
+                            background: '#AFC3D9',
+                            borderRadius: '32px'
+                        }}
+                    >
+                        <Box
+                            sx={{
+                                display: 'flex',
+                                justifyContent: 'space-between'
+
+                            }}
+                        >
+                            <Typography 
+                                sx={{
+                                    color: '#686868',
+                                    fontFamily: 'Open Sans',
+                                    fontSize: '20px',
+                                    fontWeight: '600'
+                                }}
+                            >
+                                Upload CSV File
+                            </Typography>
+                            <Typography
+                                 sx={{
+                                    color: '#686868',
+                                    fontFamily: 'Open Sans',
+                                    fontSize: '20px',
+                                    fontWeight: '400',
+                                    fontStyle: 'italic'
+                                }}
+                            >
+                                scholarship_template.csv
+                            </Typography>
+                        </Box>
+                            <form>
+                                    <TextField
+                                        sx={{
+                                            width: '100%',
+                                            margin: '20px 0',
+                                            padding: '20px 0',
+                                            "& fieldset": { border: 'none' },
+                                            borderWidth: "2px",
+                                            borderStyle: "dashed",
+                                            borderRadius: '16px',
+                                            background: '#fff',
+                                            textAlign: 'center'
+                                        }}
+                                    
+                                    type="file" />
+                                    <Button variant="contained" color="primary" component="span"
```

Comment:
Pull latest develop and use this instead
```
<Button
              variant="contained"
              component="span"
              sx={primaryButtonStyle}
              fullWidth
            >
              Save Scholarship
            </Button>
```

make sure to import primaryButtonStyle